scipy/spatial/transform/rotation.pyx

```diff
@@ -1016,6 +1019,105 @@ cdef class Rotation(object):
         else:
             return cls(quat, normalize=False, copy=False)
 
+    @classmethod
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    def from_mrp(cls, mrp):
+        """Initialize from Modified Rodrigues Parameters (MRPs).
+
+        MRPs is a 3 dimensional vector co-directional to the axis of rotation and whose
+        magnitude is equal to tan(theta / 4), where theta is the angle of rotation (in radians) [1].
+
+        MRPs have a singuarity at 360 degrees which can be avoided by ensuring the angle of
```

Comment:
This paragraph is written very solid now.

Optionally have -> has, because before we referred to MRPs as a singular entity (which is OK to my understanding). But I'm not super strong in grammar anyhow.

Reply:
Good point, I actually changed "is" to "are" in the above paragraphs. "MRPs" is plural, so I think it makes sense to treat it as such. There are 3 of them after all...